src/app/components/PageLayout/Header.tsx

```diff
@@ -1,16 +1,21 @@
 import { FC } from 'react'
 import Grid from '@mui/material/Unstable_Grid2'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { useTheme } from '@mui/material/styles'
 import { Logotype } from './Logotype'
 import { NetworkHeader } from './NetworkHeader'
 
 export const Header: FC = () => {
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+
   return (
     <header>
-      <Grid container sx={{ px: 6, pb: 5 }}>
-        <Grid xs={12} sx={{ pb: '50px' }}>
+      <Grid container sx={{ px: isMobile ? 4 : 6, pb: isMobile ? 4 : 5 }}>
+        <Grid md={12} xs={6} sx={{ pb: isMobile ? 0 : '50px' }}>
```

Comment:
This that this would be better if it would be moved to styled component. Can be left for now, we can make a clean refactor once.

Reply:
I mean `isMobile ? 4 : 6` stuff, which makes it hard to read.

Reply:
good idea, another is to split components Header vs MobileHeader etc to avoid all these crap. I will refactor this after filling "Search Placeholder" gap.

Reply:
Depends on how much diff there is between the 2. As having 2 separate components would just make our bundle increase. And you need an `if` somewhere anyway to distinguish between them.